pyproject.toml

```diff
@@ -27,9 +27,8 @@ Source = "https://github.com/posit-dev/posit-sdk-py"
 Issues = "https://github.com/posit-dev/posit-sdk-py/issues"
 
 [tool.pytest.ini_options]
-python_files = [
-    "*_test.py",
-    "*_test_*.py",
+addopts = [
+    "--import-mode=importlib",
```

Comment:
Have you verified that this gives you the functionality you want?  

https://docs.pytest.org/en/7.1.x/explanation/pythonpath.html?highlight=import%20mode#import-modes

> One drawback however is that test modules are non-importable by each other. Also, utility modules in the tests directories are not automatically importable because the tests directory is no longer added to [sys.path](https://docs.python.org/3/library/sys.html#sys.path).

If I understand this correctly, this means that abstracting mocked classes into their own module won't work.

Reply:
I have not verified. I was just blindly following the pytest docs recommendation: "For new projects, we recommend to use importlib import mode." Happy to remove if/when it's a problem.